packages/analytics-js-integrations/src/integrations/Webengage/browser.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable class-methods-use-this */
+import { logger } from '@rudderstack/analytics-js-common/v1.1/utils/logUtil';
```

Comment:
use the new logger as per the https://github.com/rudderlabs/rudder-sdk-js/pull/1434 PR

Reply:
same goes for all logger calls, follow same conventions on how logger is utilised